<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QMainWindow" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>767</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>926</width>
    <height>767</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>926</width>
    <height>767</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Euler</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget {
    background-color: black;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="8" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="3">
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="11" column="3">
       <spacer name="verticalSpacer_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="3">
       <spacer name="verticalSpacer_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="3">
       <widget class="QRadioButton" name="radioVertices">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>55</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton#radioVertices:indicator {
    width: 50px;
    height: 50px;
}
QRadioButton#radioVertices:pressed {
    background-color: rgb(39, 209, 255);
}
QRadioButton#radioVertices {
    background-color: white;
    padding-left: 10px;
}
</string>
        </property>
        <property name="text">
         <string>Edit vertices</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QRadioButton" name="radioEdges">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>55</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton#radioEdges:indicator {
    width: 50px;
    height: 50px;
}
QRadioButton#radioEdges:pressed {
    background-color: rgb(39, 209, 255);
}
QRadioButton#radioEdges {
    background-color: white;
    padding-left: 10px;
}
</string>
        </property>
        <property name="text">
         <string>Edit edges</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QPushButton" name="pushClear">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>55</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pushClear {
    background-color: white;
    padding: 3px;
    border-radius: 10px;
}
QPushButton#pushClear:pressed {
     background-color: rgb(39, 209, 255);
     border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QPushButton" name="pushStop">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>55</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pushStop {
    background-color: white;
    padding: 3px;
    border-radius: 10px;
}
QPushButton#pushStop:pressed {
     background-color: rgb(39, 209, 255);
     border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item row="14" column="3">
       <widget class="QPushButton" name="pushPrevious">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>55</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pushPrevious {
    background-color: white;
    padding: 3px;
    border-radius: 10px;
}
QPushButton#pushPrevious:pressed {
     background-color: rgb(39, 209, 255);
     border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>Previous</string>
        </property>
       </widget>
      </item>
      <item row="18" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="3">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1" rowspan="15">
       <widget class="QGraphicsView" name="graphicsView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>633</width>
          <height>423</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;</string>
        </property>
        <property name="interactive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="pushSolve">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>55</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pushSolve {
    background-color: white;
    padding: 3px;
    border-radius: 10px;
}
QPushButton#pushSolve:pressed {
     background-color: rgb(1, 139, 170);
     border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>Solve</string>
        </property>
       </widget>
      </item>
      <item row="12" column="3">
       <widget class="QPushButton" name="pushNext">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>55</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pushNext {
    background-color: white;
    padding: 3px;
    border-radius: 10px;
}
QPushButton#pushNext:pressed {
     background-color: rgb(39, 209, 255);
     border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
      <item row="13" column="3">
       <spacer name="verticalSpacer_9">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background-color: black;
color: white;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
